src/prompts/alert/index.tsx

```diff
@@ -0,0 +1,74 @@
+import './alert.scss';
+
+import { Component, h } from 'preact';
+import { PromptUiProps } from "../ui";
+
+export class Alert extends Component<PromptUiProps, never> {
+  onRequestNativePrompt = () => {
+      this.props.onPromptAccepted(this.props.config);
+  };
+
+  onRequestCancel = () => {
+      this.props.onPromptDeclined(this.props.config);
+  };
+
+  render() {
+      const classes = `kumulos-prompt kumulos-prompt-${this.props.promptManagerState} kumulos-alert-container`;
+      const config = this.props.config;
+
+      if (config.type !== 'alert') {
```

Comment:
Can we strategise / generify PromptUiProps and make the config a specific AlertPromptCfg type?

Reply:
Done 👍